types/query.d.ts

```diff
@@ -95,33 +95,70 @@ declare module 'mongoose' {
     updatedAt?: boolean;
   }
 
+  interface MongooseSpecificQueryOptions {
+    /**
+     * If truthy, mongoose will return the document as a plain JavaScript object rather than a mongoose document.
+     */
+    lean?: boolean | Record<string, any>;
+
+    multipleCastError?: boolean;
+
+    overwriteDiscriminatorKey?: boolean;
+    runValidators?: boolean;
```

Comment:
```suggestion
    /**
     * Set to true to enable `update validators`
     * (https://mongoosejs.com/docs/validation.html#update-validators). Defaults to false.
     */
    runValidators?: boolean;
```

missing tsdoc that was previously there?